src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -113,6 +113,10 @@
             optional = true)
     public boolean disableBamIndexCaching = false;
 
+    @Argument(fullName = StandardArgumentDefinitions.SITES_ONLY_LONG_NAME,
+            doc = "If true, don't emit GT fields when writing vcf file output.", optional = true)
+    public boolean sitesOnlyMode = false;
```

Comment:
Name of field should make it clear that this applies to VCFs specifically -- eg., `outputSitesOnlyVCFs` instead of `sitesOnlyMode`